### ISSUE TITLE
Image size

### ISSUE THREAD

**Issue by YoadTew** [AlexeyAB/darknet#2331]
Hey @AlexeyAB ,

thanks for all the explanation and great repo.
I want to use yolo to detect costume objects in an image of size 240x80 (width=240, height=80).
(More specific, I want to detect and recognize numbers (10 classes) in a crop of a license plate)

Im following: https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects

but I don't know if I need to change something in the image size in the cfg (Im afraid that the image resizing to different scale will affect performance).
Also, will it help to do something with the anchors? because all my boxes would be around 2:1 height to width ratio.

Thank you!

**Comment by AlexeyAB** [comment]
@YoadTew Hi,
You can set `width=256 height=96` in your cfg-file.
And use default anchors.

**Comment by YoadTew** [comment]
@AlexeyAB Thanks you for the answer!

I'm asking because I'm not sure yet,
wouldn't it be better if the anchors are chosen specifically for the task?
Because I know the shapes almost always will be 1:2 ratio.

**Comment by AlexeyAB** [comment]
@YoadTew Yes, you can do it. But follow the conditions below https://github.com/AlexeyAB/darknet#how-to-improve-object-detection

> recalculate anchors for your dataset for width and height from cfg-file: darknet.exe detector calc_anchors data/obj.data -num_of_clusters 9 -width 416 -height 416 then set the same 9 anchors in each of 3 [yolo]-layers in your cfg-file
> 
> **(also you can change indexes of anchors masks= for each [yolo]-layer, so that 1st-[yolo]-layer has anchors larger than 60x60, 2nd larger than 30x30, 3rd remaining)**
